### PR TITLE
All loaders moved to denali

### LINKED ISSUE #1210: Move all loaders to denali

### PR DESCRIPTION
**Pull request by holderb5**
Resolves #1210 

I believe all of the loaders are working well, but I'd like someone to open this branch and double check them if they have time.
Screen Shot 2021-01-27 at 1 31 19 PM
Screen Shot 2021-02-01 at 9 58 17 AM

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jkusa** [comment]
@holderb5 can you please add screenshots?

**Comment by jkusa** [comment]
@holderb5 looks like the loaders need to be centered.

File: packages/dashboards/addon/templates/common/navi-loader.hbs [hunk 1]
```diff
@@ -1,6 +1,2 @@
 {{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<div class="spinner">
-  <div class="bounce1"></div>
-  <div class="bounce2"></div>
-  <div class="bounce3"></div>
-</div>
+<DenaliLoader />
```
Comment (jkusa) [inline packages/dashboards/addon/templates/common/navi-loader.hbs:2]: Hmm, can we remove this one. I am not sure this is used.

File: packages/core/app/styles/navi-core/components/navi-loader.less [hunk 2]
```diff
@@ -7,12 +7,14 @@
   &__container {
     display: flex;
     flex-flow: column;
+    align-items: center;
+    justify-content: center;
     margin: 10px;
     width: 100%;
+    height: 300px;
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/core/app/styles/navi-core/components/navi-loader.less:14]: To fix the directory table centering, you can add the class `h-50` (denali utility class, height: 250px) to the loader component: 
https://github.com/yahoo/yavin/blob/master/packages/directory/addon/components/dir-table.hbs#L23
```hbs
        <NaviLoader class="h-50"/>
 ```

File: packages/dashboards/addon/templates/common/navi-loader.hbs (hunk 1 above)
Comment (holderb5) [reply packages/dashboards/addon/templates/common/navi-loader.hbs:2]: Doesnt seem to be used, I removed it and ran the dashboards tests with no fails

File: packages/core/tests/integration/components/navi-loader-test.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ module('Integration | Component | navi loader', function(hooks) {
   setupRenderingTest(hooks);
 
   test('it renders', async function(assert) {
-    assert.expect(4);
+    assert.expect(3);
 
     await render(hbs`{{navi-loader
```
Comment (jkusa) [inline packages/core/tests/integration/components/navi-loader-test.js:12]:
```suggestion
    await render(hbs`<NaviLoader class="test-container" />`);
```

**Review by jkusa — APPROVED** [review]

**Pull request merged**